File: buff/classes/354/pub/etc/Sum.cs
// C# OO sum program

public class Sum {

    private int s;

    public Sum() {
	init();
    }

    public void init() {
	s=0;
    }

    public int sum(int[] seq) {
	foreach (int v in seq)
	    s+=v;
	return s;
    }

    public static void Main() {
	int[] seq=new int[] {5,6,1,8,3,7};
	Sum sum=new Sum();
	System.Console.WriteLine(sum.sum(seq));
    }

}

File: buff/classes/354/pub/sum/armv8-a/sum.c
/* C sum program */

#include <stdio.h>

static int sum(int seq[], int n) {
  int s=0;
  int i;
  for (i=0; i<n; i++)
    s+=seq[i];
  return s;
}

int main() {
  const int n=6;
  int seq[]={5,6,1,8,3,7};
  printf("%d\n",sum(seq,n));
  return 0;
}
